pipelines/ci-kubernetes-pipeline.yml

```diff
@@ -83,13 +96,63 @@ resources:
     username: _json_key
     password: ((gcp-service-account-json))
 
+
 jobs:
 
+
+- name: deploy-infrastructure
+  serial: true
+  serial_groups: [action-scheduler, actionexportersvc, case-processor, iacsvc, pubsubsvc]
+  max_in_flight: 1
+  plan:
+  - get: census-rm-terraform
+    trigger: true
+  - get: census-rm-kubernetes-repo
+  - task: "Build & Deploy Infrastructure"
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: google/cloud-sdk
+      params:
+        AUTO_APPLY: true
+        DEV_PUBSUB: true
+        DEV_SCALE_DOWN: true
+        ENV: ((gcp-environment-name))
+        GOOGLE_APPLICATION_CREDENTIALS: /root/gcloud-service-key.json
+        KEEP_TMP_K8S: true
+        SERVICE_ACCOUNT_JSON: ((gcp-admin-account-json))
+      inputs:
+        - name: census-rm-terraform
+        - name: census-rm-kubernetes-repo
+      run:
+        path: bash
+        args:
+          - -exc
+          - |
+            git clone https://github.com/kamatama41/tfenv.git ~/.tfenv && \
```

Comment:
Is this safe to do? I know eQ do the same but maybe ONSDigital should fork this to guard against malicious future commits.

Reply:
seems like a reasonable precaution to fork